build-container.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
1) if we are going to have developer scripts, we should put them in a scripts directory
2) i think this would be better encoded into a makefile with targets like "make images" that do this sort of thing (even if the make target then calls "scripts/build-container.sh")

Reply:
so `makefile` in the project root, and then a `scripts` folder that houses things that aren't directly used by the app, and then all app stuff under `src`?

Reply:
yeah that seems like a good fundamental structure